class/defaults.yml

```diff
@@ -1,4 +1,8 @@
 parameters:
+  global:
+    http_proxy: ""
+    https_proxy: ""
+    no_proxy: ""
```

Comment:
We generally have all the component's parameters in key `parameters.<component_name>` as there's no clear definition who "owns" keys in `parameters` which don't correspond to a component name.